File: smiex_api/smiex_api/Models/Documentacion.cs
using System;
using System.Collections.Generic;

namespace smiex_api.Models;

public partial class Documentacion
{
    public int IdDocumento { get; set; }

    public string NombreDoc { get; set; } = null!;

    public string UrlArchivo { get; set; } = null!;

    public int IdOferta { get; set; }

    public virtual Oferte IdOfertaNavigation { get; set; } = null!;
}

File: simex_app/app/src/main/java/com/example/simex_app/data/models/Documentacion.kt
package com.example.simex_app.data.models

data class Documentacion(
    val idDocumento: Int,
    val nombreDoc: String,
    val urlArchivo: String,
    val idOferta: Int,

    // Relación con la Oferta (Navegación)
    // Se pone como opcional (?) para que no explote si el JSON no la incluye
    val idOfertaNavigation: Oferte? = null
)